### DISCUSSION TITLE
[solved] THS317-ET Issues with some Firmware versions?

### DISCUSSION THREAD

**Discussion by woessmich** [Koenkk/zigbee2mqtt#15602] [Devices]
Hi,

I have two THS317-ET, one works fine since months, the other one pairs fine, but then shows as not connected and I never get data. It unpairs and pairs everytime I try, though. I sent it back, received a new one and same problem with it, so probably not a dfective device.

The one working fine reports as:
date code: 20211101 and build V1.0.7

The not working one reports as:
date code: 20221019 but no build is shown

During pairing I see this error messages:
`New device joined '0x3c6a2cfffed2f7b8' model THS317-ET
Error: DeviceConfigure:Failed to DeviceConfigure.configure 0x3c6a2cfffed2f7b8 THS317-ET (TypeError: Cannot read properties of undefined (reading 'bind')
    at Object.bind (/opt/iobroker/node_modules/zigbee-herdsman-converters/lib/reporting.js:40:24)
    at Object.configure (/opt/iobroker/node_modules/zigbee-herdsman-converters/devices/owon.js:185:29)
    at DeviceConfigure.doConfigure (/opt/iobroker/node_modules/iobroker.zigbee/lib/zbDeviceConfigure.js:146:32)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at DeviceConfigure.configure (/opt/iobroker/node_modules/iobroker.zigbee/lib/zbDeviceConfigure.js:124:17)). undefined
Interview successful: {"friendly_name":"0x3c6a2cfffed2f7b8","model":"THS317-ET","vendor":"OWON","description":"Temperature sensor","supported":true} `

I am using the iobroker implementation of zigbee2MQTT.
I have seen other discussions around this specific device, but they look like different issues.
Any idea what could be a solution?
Thank you!

**Comment by woessmich** [comment]
I did some more testing with pairing, unpairing etc. same problem every time.
But, the Sensor connects and shows temperature with an Alexa Echo. So, something might be wrong with the configuration of the zigbee receiving end.

**Comment by woessmich** [comment]
Hi
I just updated the iobroker zigbee Adapter to version 1.8.9, unpaired the sensor and paired it again. Now it interviewed without error and I successfully receive temperatures.

**Comment by okovjogurtu** [comment]
Exactly same issue here, 2 device with 20211101and one with 20221019.
Zigbee2mqtt 1.32.1
zStack3x0, 20230507 (same problem with previous ver.) Sonoff zigbee usb dongle

**Comment by lecourtb** [comment]
Same problem wit z2m
